build/webpack.config.js

```diff
@@ -78,6 +83,20 @@ function updateConfig(conf, language, index) {
         if (rsyncLocation) {
             config.plugins.push(new RsyncPlugin('dist/.', rsyncLocation));
         }
+
+        if (rsyncApiLocation) {
```

Comment:
Should this be an else if? Is there a circumstance where we would need to support both cases?

Reply:
I don't see a use case right now for syncing to a API based and non API based host at the same time but left it supported. I imagine the normal use case would be either or though so I could update. 

Should I leave it as is or support both?